### PR TITLE
Add conduit example app

### PR DESCRIPTION
**Pull request by sldk-yuri**
## Background
This example is my implementation of Conduit [(RealWorld example apps)](https://github.com/gothinkster/realworld) write on Feature-Sliced Design architectural methodology. It's already added to RealWorld showcases, and I think it would be fit in FSD examples as well. Could you take a look at the project and do a review, please?

- source: https://github.com/sldk-yuri/realworld-react-fsd
- website: https://realworld-fsd.netlify.app/

Thank you.

### CODE REVIEW

**Review by azinit — CHANGES_REQUESTED** [review]
I review project basically, but it looks great already! I suppose one change and after it we could merge your example in our FSD list

Thank you for contribution! 🔥

File: src/pages/examples/_config.ts [hunk 1]
```diff
@@ -181,5 +181,16 @@ export const examples: Example[] = [
         updatedAt: "2023-06-08",
         tech: ["react", "effector", "typescript", "vite", "tailwind", "atomic-router"],
     },
+    {
+        title: "Conduit",
+        description:
+            "A social blogging site powered by Feature-Sliced Design architectural methodology.",
+        website: "https://realworld-fsd.netlify.app/",
+        source: "https://github.com/sldk-yuri/realworld-react-fsd",
```
[the remark this answers is not in the document]
Comment (sldk-yuri) [reply src/pages/examples/_config.ts:189]: @azinit sldk-yuri/realworld-react-fsd@e3675a48fb3656c0c681651b2b897f852b0081c1 page folders structure is updated. Thank you for the review. 🙏

**Comment by sldk-yuri** [comment]
@azinit Could you update `updatedAt` date in `_config.ts` before merge, please? Thank you.

File: src/pages/examples/_config.ts [hunk 2]
```diff
@@ -181,5 +181,16 @@ export const examples: Example[] = [
...
+        preview: require("./img/conduit.png"),
+        version: VERSIONS.V2,
+        updatedAt: "2023-06-15",
```
Comment (azinit) [inline src/pages/examples/_config.ts:192]:
```suggestion
        updatedAt: "2023-06-21",
```

**Review by azinit — APPROVED** [review]

**Comment by azinit** [comment]
@sldk-yuri Thanks for your contribution! ❤️

**Pull request merged**